checkov/sca_package/output.py

```diff
@@ -0,0 +1,294 @@
+
+            if package_name in package_details_map.keys():
+                package_details_map[package_name]["cves"].sort(key=compare_cve_severity)
+                package_details_map[package_name]["current_version"] = package_version
+                package_details_map[package_name]["complaint_version"] = calculate_lowest_complaint_version(
+                    fix_versions_lists
+                )
+
+        cli_outputs.append(
+            create_cli_table(
+                file_path=file_path,
+                cve_count=cve_count,
+                package_details_map=package_details_map,
+            )
+        )
+
+    return "".join(cli_outputs)
+
+
+def create_cli_table(file_path: str, cve_count: Dict[str, int], package_details_map: Dict[str, Dict[str, Any]]) -> str:
+    columns = 6
+    table_width = 120
+    column_width = 120 / columns
+    cve_table = PrettyTable(
+        header=False,
+        padding_width=1,
+        min_table_width=table_width,
+        max_table_width=table_width,
+    )
+    cve_table.set_style(SINGLE_BORDER)
+    cve_table.add_row(
+        [
+            f"Total CVEs: {cve_count['total']}",
+            f"critical: {cve_count['critical']}",
+            f"high: {cve_count['high']}",
+            f"medium: {cve_count['medium']}",
+            f"low: {cve_count['low']}",
+            f"skipped: {cve_count['skipped']}",
+        ]
+    )
+    cve_table.align = "l"
+    cve_table.min_width = column_width
+    cve_table.max_width = column_width
+
+    cve_table_lines = [f"\t{line}" for line in cve_table.get_string().splitlines(keepends=True)]
+    # hack to make multiple tables look like one
+    cve_table_bottom_line = (
+        cve_table_lines[-1]
+        .replace(cve_table.bottom_left_junction_char, cve_table.left_junction_char)
+        .replace(cve_table.bottom_right_junction_char, cve_table.right_junction_char)
+    )
+    cve_table_lines[-1] = cve_table_bottom_line
+
+    fixable_table = PrettyTable(
+        header=False, min_table_width=table_width + columns * 2, max_table_width=table_width + columns * 2
+    )
+    fixable_table.set_style(SINGLE_BORDER)
+    fixable_table.add_row(
+        [f"To fix {cve_count['fixable']}/{cve_count['to_fix']} CVEs, go to https://www.bridgecrew.cloud/"]
+    )
+    fixable_table.align = "l"
+
+    # hack to make multiple tables look like one
+    fixable_table_lines = [f"\t{line}" for line in fixable_table.get_string().splitlines(keepends=True)]
+    del fixable_table_lines[0]
+    # only remove the last line, if there are vulnerable packages
+    if package_details_map:
+        del fixable_table_lines[-1]
+
+    package_table_lines: List[str] = []
+    package_table = PrettyTable(min_table_width=table_width, max_table_width=table_width)
+    package_table.set_style(SINGLE_BORDER)
+    package_table.field_names = [
+        "Package",
+        "CVE ID",
+        "Severity",
+        "Current version",
+        "Fixed version",
+        "Complaint version",
```

Comment:
```suggestion
        "Compliant version",
```

Reply:
when you do it wrong, then at least it is consistent 😂